src/Makefile.boot

```diff
@@ -2,7 +2,6 @@ include Makefile.config
 
 # FSTAR_C: This is the way in which we invoke F* for boostrapping
 #   -- we use automatic dependence analysis based on files in ulib, src/{basic, ...} and boot
-#   -- eager_inference, MLish, lax: all tune type-inference for use with unverified ML programs
```

Comment:
We still use `--MLish` and `--lax`, so I wouldn't remove this entire line, but just the `eager_inference` part. Also, while you're at it, could you move this entire comment to the definition of `FSTAR_C`? That's what it's speaking about, but it seems to have drifted away :).

Reply:
Fixed.